ferolino_jasmin/country.html

```diff
@@ -8,15 +8,15 @@
 </head>
 <body>
     <h1>Countries of the World</h1>
-    <input type="text" id="countryInput" placeholder="Enter a country">
+    <input type="text" id="country_input" placeholder="Enter a country">
     <button onclick="searchCountry()">Search</button>
 
-    <div id="countryDetails">
+    <div id="countrydetails">

```

Comment:
### IDs should be written in snake case. All lower case separated by underscore.

This the appropriate id: country_details

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

This also applies in line 17